checker/check.go

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) 2018 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package checker
+
+import (
+	authz "github.com/envoyproxy/data-plane-api/api/auth"
+
+	"github.com/projectcalico/app-policy/policystore"
+
+	"github.com/projectcalico/app-policy/proto"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/genproto/googleapis/rpc/code"
+	"google.golang.org/genproto/googleapis/rpc/status"
+	"strings"
+)
+
+var OK = code.Code_value["OK"]
+var PERMISSION_DENIED = code.Code_value["PERMISSION_DENIED"]
+
+type Action int
+
+const (
+	ALLOW Action = iota
+	DENY
+	LOG
+	PASS
+	NO_MATCH // Indicates policy did not match request. Cannot be assigned to rule.
+)
+
+// Check a list of policies and return OK if the check passes, or PERMISSION_DENIED if the check fails.
+// Note, if no policy matches, the default is PERMISSION_DENIED.
+func checkStore(store *policystore.PolicyStore, req *authz.CheckRequest) (s status.Status) {
```

Comment:
Go best practice is to start the comment with the func/field name itself.
So something like: `// checkStore checks a list of ...`

Reply:
+1